### DISCUSSION TITLE
How do I configure Svelte Kit to pull static data from a CDN?

### DISCUSSION THREAD

**Discussion by shackra** [sveltejs/kit#5403] [Q&A] [answered]
I'm trying to figure out how to accomplish that. I would like to serve whatever is in `static/` after building the project from a CDN.

My issue is that I have no experience doing this, and you can hardly find any code/configuration examples or tutorials on the subject.

So, assuming I already have the contents of `static/` uploaded somewhere (or in a CDN), how should I configure Svelte Kit for working like this in production?

**Comment by benmccann** [comment, marked as answer]
Use `paths.assets`. See https://kit.svelte.dev/docs/configuration#paths
